taichi/program/compile_config.h

```diff
@@ -11,6 +11,7 @@ struct CompileConfig {
   bool debug;
   bool cfg_optimization;
   bool check_out_of_bound;
+  bool check_autodiff_valid;
```

Comment:
```suggestion
  bool validate_autodiff;
```

Reply:
Yes. The final solution is controlled in the `ti.ad.Tape`.